server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java

```diff
@@ -253,15 +259,17 @@ public final class JettyServerConfig {
   public static final ConfigEntry<String> EXPOSED_HEADERS =
       new ConfigBuilder("exposedHeaders")
           .doc(
-              "A comma separated list of HTTP headers that are allowed to be exposed on the client. Default value is the empty list")
+              "A comma separated list of HTTP headers that are allowed to be exposed on the client."
```

Comment:
I think you miss a whitespace when you fold the sentence into two lines. Not just here, but also in the above code.